Godeps/Godeps.json

```diff
@@ -18,6 +18,11 @@
 			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
 		},
 		{
+			"ImportPath": "github.com/BurntSushi/toml",
```

Comment:
Thanks for reviewing, I have asked the developer associated with the code using this dep to investigate alternatives